lib/p2p/Peer.ts

```diff
@@ -173,7 +174,7 @@ class Peer extends EventEmitter {
   /**
    * @param address The socket address for the connection to this peer.
    */
-  constructor(private logger: Logger, public address: Address, network: Network) {
+  constructor(private logger: Logger, public address: Address, network: Network, private torSocks?: number) {
```

Comment:
I don't think we need to make `torSocks` a member field for each peer, right? I think it makes more sense to be passed into the `beginOpen` method, then onto the `initConnection` method where it's used.

Reply:
You're right, good thinking. Changed.